care/facility/tests/test_patient_consultation_api.py

```diff
@@ -1,3 +1,5 @@
+from rest_framework import status
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5380357)